src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.Android.cs

```diff
@@ -18,6 +13,15 @@ public sealed partial class StatusBar
 		private StatusBarForegroundType? _foregroundType;
 		private bool? _isShown;
 
+		private static readonly float _cachedDensity;
```

Comment:
We already cache it here:

https://github.com/unoplatform/uno/blob/30d27c96948d09ef6e355e2d488e38f434ac3811/src/Uno.UI/Extensions/ContextExtensions.Android.cs#L32-L42

I think we should have a central place where we cache it, and update all places where we access Density.

Reply:
Actually, we have such central place:


https://github.com/unoplatform/uno/blob/30d27c96948d09ef6e355e2d488e38f434ac3811/src/Uno.UI/Extensions/ViewHelper.Android.cs#L43-L52

https://github.com/unoplatform/uno/blob/30d27c96948d09ef6e355e2d488e38f434ac3811/src/Uno.UI/Extensions/ViewHelper.Android.cs#L77-L99

Reply:
We should be using `DisplayInformation` instead, which should already provides the proper caching (particularly if the density changes between screens).